pensando_psm/info.json

```
@@ -0,0 +1,544 @@
+{
+    "name": "pensando_psm",
+    "version": "1.0.0",
+    "label": "Pensando Policy Service Manager",
+    "description": "Perform actions on Pensando Policy Service Manager",
+    "publisher": "Fortinet",
+    "icon_small_name": "pensando_logo_small.jpg",
+    "icon_large_name": "pensando_logo_large.jpg",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "Server Address",
+                "type": "text",
+                "name": "server_address",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Server Port",
+                "type": "text",
+                "name": "port",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "443"
+            },
+            {
+                "title": "Tenant",
+                "type": "text",
+                "name": "tenant",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "default"
+            },
+            {
+                "title": "Username",
+                "type": "text",
+                "name": "username",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Password",
+                "type": "password",
+                "name": "password",
+                "required": true,
+                "visible": true,
+                "editable": true
+            },
+            {
+                "title": "Protocol",
+                "type": "select",
+                "name": "protocol",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "HTTPS",
+                "options": [
+                    "HTTP",
+                    "HTTPS"
+                ]
+            },
+            {
+                "title": "Verify SSL",
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "required": false,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "debug_remove_session_state",
+            "title": "debug - remove session state",
+            "description": "Debug login by removing session state from disk.",
+            "enabled": false,
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            }
+        },
+        {
+            "operation": "debug_reset_session_state",
+            "title": "debug - reset session state",
```

```suggestion
            "title": "Debug - Reset Session State",
```
Please correct the title with camel case.

---

This is a test operation that will be deleted.